src/RestApi/StoreApi/Schemas/CustomerSchema.php

```diff
@@ -0,0 +1,195 @@
+						'type'        => 'string',
+						'format'      => 'email',
+						'context'     => [ 'view', 'edit' ],
+						'arg_options' => [
+							'sanitize_callback' => 'sanitize_email',
+							'validate_callback' => 'is_email',
+						],
+					],
+					'phone'      => [
+						'description' => __( 'The billing contact number of the customer.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+				],
+			],
+			'shipping' => [
+				'description' => __( 'List of shipping address data.', 'woo-gutenberg-products-block' ),
+				'type'        => 'object',
+				'context'     => [ 'view', 'edit' ],
+				'properties'  => [
+					'first_name' => [
+						'description' => __( 'First name of the customer for the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'last_name'  => [
+						'description' => __( 'Last name of the customer for the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'company'    => [
+						'description' => __( 'Company name for the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'address_1'  => [
+						'description' => __( 'First line of the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'address_2'  => [
+						'description' => __( 'Second line of the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'city'       => [
+						'description' => __( 'City of the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'state'      => [
+						'description' => __( 'ISO code, or name, for the state, province, or district of the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'postcode'   => [
+						'description' => __( 'Zip or Postcode of the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+					'country'    => [
+						'description' => __( 'ISO country code for the shipping address.', 'woo-gutenberg-products-block' ),
+						'type'        => 'string',
+						'context'     => [ 'view', 'edit' ],
+					],
+				],
+			],
+		];
+	}
+
+	/**
+	 * Convert a woo customer into an object suitable for the response.
+	 *
+	 * @param CustomerObject $object Customer object.
+	 * @return array
+	 */
+	public function get_item_response( $object ) {
+		return [
+			'id'       => is_user_logged_in() ? $object->get_id() : 0,
```

Comment:
Can’t we use `get_current_user_id()`

Reply:
Maybe just use get_id() from the object so it reflects whatever is passed?